docs/contributing/building.rst

```diff
@@ -153,6 +153,22 @@ or source it into your shell,
 
    . $BUILD_DIR/python_env.sh
 
+PyTorch
+=======
+
+To install PyTorch, follow the commands below. The first 2 lines install dependencies. The subsequent steps install PyTorch from source.
+
+.. code-block:: bash
+
+   mamba install -c conda-forge --override-channels astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses
+   mamba install -c pytorch magma-cuda110
```

Comment:
My understanding was that we were still not sure about advocating the use of mamba. @arthurp is this still correct?

Reply:
Sorry, I changed all conda calls to mamba in my notes. For this documentation, it's now conda.